pkg/lib/cockpit-po-plugin.js

```diff
@@ -24,15 +25,17 @@ module.exports = class {
         if (!header)
             return null;
 
-        var body, statement, ret = null;
+        var statement;
+        var ret = null;
         const plurals = header["plural-forms"];
 
         if (plurals) {
             try {
                 /* Check that the plural forms isn't being sneaky since we build a function here */
                 Jed.PF.parse(plurals);
-            } catch(ex) {
-                fatal("bad plural forms: " + ex.message, 1);
```

Comment:
I don't know where `fatal` would come from.  Is it a real thing that we can call?

Reply:
It used to be defined in `po/po2json` as:
```
function fatal(message, code) {                                                 
    console.log((filename || "html2po") + ": " + message);                      
    process.exit(code || 1);                                                    
} 
```
See https://github.com/cockpit-project/cockpit/commit/72c180d78848946b939bbab23b063eee246336d8

Reply:
Oops! Thanks for catching that!